src/context/MyContext.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from "react";
+
+export const MyContext = createContext("");
```

Comment:
Rename this context. Name should give instinct about what is this context about

Reply:
Context rename completed.